keel-sql/src/main/resources/db/changelog/20191202-create-diff-fingerprint-table.yml

```diff
@@ -0,0 +1,48 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-diff-fingerprint-table
+      author: emjburns
+      changes:
+        - createTable:
+            tableName: diff_fingerprint
+            columns:
+              - column:
+                  name: resource_id
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: first_detection_time
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: count
+                  type: int
+                  defaultValue: 1
+                  constraints:
+                    nullable: false
+              - column:
+                  name: hash
+                  type: longtext
```

Comment:
This can be fixed length, based on the hash algo used. For sha1, I think `char(40)` is appropriate.